src/lib/CMakeLists.txt

```diff
@@ -592,7 +592,7 @@ set(
     sqlparser
     cqf
     uninitialized_vector
-    jemalloc
+    custom_jemalloc
```

Comment:
This doesn't really *change* anything, but I wanted to make it clear that we are using our own one.